.build.yml

```diff
@@ -212,11 +212,7 @@ before_scripts:
   -     git clone --depth 1  https://github.com/AyatanaIndicators/libayatana-common.git libayatana-common-build
   - fi
   - cd libayatana-common-build
-  - if [ ${DISTRO_NAME} == "debian" ]; then
-  -     cmake . -DCMAKE_INSTALL_PREFIX=/usr -DENABLE_LOMIRI_FEATURES=ON
-  - else
-  -     cmake . -DCMAKE_INSTALL_PREFIX=/usr
-  - fi
+  - cmake . -DCMAKE_INSTALL_PREFIX=/usr
```

Comment:
Urgh, why do you remove -DENABLE_LOMIRI_FEATURES=ON for libayatana-common? This should stay IMHO.

Reply:
> Urgh, why do you remove -DENABLE_LOMIRI_FEATURES=ON for libayatana-common? This should stay IMHO.

What the?! Sorry, cosmic radiation/sunspots, probably